Create acosh entry and test all changes (#1284)
Codecademy/docs: content/cpp/concepts/math-functions/terms/acosh/acosh.md

---
Title: 'acosh()'
Description: 'Returns the arc hyperbolic cosine of an angle given in radians.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

The **`acosh()`** function returns the arc hyperbolic cosine of an angle given in radians.

## Syntax

```pseudo
acosh(n)
```

> The `n` parameter can be a floating-point value (e.g., a `long` or `double`) or an integral type (e.g., an `int` or `bool`). `n` is mandatory and must be greater or equal to 1. If it is less than 1, a domain error occurs.

Reviewer: ```suggestion
The `n` parameter can be a floating-point value (e.g., a `long` or `double`) or an integral type (e.g., an `int` or `bool`). `n` is mandatory and must be greater or equal to 1. Values less than 1 will result in a domain error.
```

## Example

This example uses the `acosh()` function to return the arc hyperbolic cosine of `1`:

```cpp
#include <iostream>
#include <cmath>

int main() {
  double x = 1;
  double result;
  result = std::acosh(x);
  std::cout << result << " radian" << "\n";
}
```

> This example results in the following output:

Reviewer: ```suggestion
This results in the following output:
```
